naga/src/front/wgsl/parse/number.rs

```diff
@@ -16,12 +18,23 @@ pub enum Number {
     I64(i64),
     /// Concrete u64
     U64(u64),
+    /// Concrete f16
+    F16(f16),
     /// Concrete f32
     F32(f32),
     /// Concrete f64
     F64(f64),
 }
 
+impl Number {
+    pub(super) const fn required_enable_extension(&self) -> Option<ImplementedEnableExtension> {
```

Comment:
nitpick: I think present indicative is usually a better tense for query methods like this than past indicative, i.e., "requires", rather than "required".

suggestion: Let's use `requires_enable_extension`.